src/main/java/de/t0biii/music/domain/ConfigManager.java

```diff
@@ -11,18 +11,24 @@ public ConfigManager(Main plugin) {
   }
 
   public void loadConfig() {
-    this.plugin.getConfig().options().header("Plugin by itzTobi_!\nChange at your own risk");
+    this.plugin.getConfig().options().header("Plugin by itzTobi_!\nChange at your own risk\nOnly the messages with color codes here support color codes. The rest have a default color");
     this.plugin.getConfig().addDefault("music", "Song.nbs");
     this.plugin.getConfig().addDefault("options.music.random", false);
     this.plugin.getConfig().addDefault("options.music.RandomFoldername", "random");
     this.plugin.getConfig().addDefault("options.music.Mode", "MonoMode");
     this.plugin.getConfig().addDefault("options.update-check", true);
     this.plugin.getConfig().addDefault("options.updateinfo", true);
-    this.plugin.getConfig().addDefault("options.printSongTitel", true);
```

Comment:
I still want this Feature in because it got requestet to dispaly nothing when someone joins the server

Reply:
When going to play music it actually first checks if the message is empty (`''`) and in that case nothing is sent, that's why I removed it, I thought that would be enough.
https://github.com/T0biii/JoinMusic/pull/27/commits/709b1e24757f137567903ef846deb5f3e62c6706#diff-6055bac5f06cb9627406485a1dd9e126R83

Reply:
Added a comment to make it obvious in the header of the config file.

Reply:
> 
> 
> When going to play music it actually first checks if the message is empty (`''`) and in that case nothing is sent, that's why I removed it, I thought that would be enough.
> [709b1e2#diff-6055bac5f06cb9627406485a1dd9e126R83](https://github.com/T0biii/JoinMusic/commit/709b1e24757f137567903ef846deb5f3e62c6706#diff-6055bac5f06cb9627406485a1dd9e126R83)

yeah i saw this. But if someone already use this option he musted edit the config file again to disable it.

Reply:
> 
> 
> Added a comment to make it obvious in the header of the config file.

i dont think the config header gets Updatet if the config exits already?
So mmh

Reply:
It does, for me it even got updated by just `/jm reload` in debug mode, without even restarting after the change.

Reply:
> It does, for me it even got updated by just `/jm reload` in debug mode, without even restarting after the change.

ok u right but i still want to keep "printSongTitel" in for backwords compatibility